diff --git a/browser/components/urlbar/UrlbarPrefs.sys.mjs b/browser/components/urlbar/UrlbarPrefs.sys.mjs
index 89f0d44d46..1bce082a99 100644
--- a/browser/components/urlbar/UrlbarPrefs.sys.mjs
+++ b/browser/components/urlbar/UrlbarPrefs.sys.mjs
@@ -163,6 +163,9 @@ const PREF_URLBAR_DEFAULTS = /** @type {PreferenceDefinition[]} */ ([
   // Feature gate pref for flight status suggestions in the urlbar.
   ["flightStatus.featureGate", false],
 
+  // Whether to close the urlbar when blurring the window while the urlbar is focused.
+  ["closeOnWindowBlur", true],
+
   // The minimum prefix length of a flight status keyword the user must type to
   // trigger the suggestion. 0 means the min length should be taken from Nimbus
   // or remote settings.
@@ -405,7 +408,7 @@ const PREF_URLBAR_DEFAULTS = /** @type {PreferenceDefinition[]} */ ([
 
   // Whether Firefox Suggest will use the new Rust backend instead of the
   // original JS backend.
-  ["quicksuggest.rustEnabled", true],
+  ["quicksuggest.rustEnabled", false],
 
   // The Suggest Rust backend will ingest remote settings every N seconds as
   // defined by this pref. Ingestion uses nsIUpdateTimerManager so the interval
@@ -482,6 +485,7 @@ const PREF_URLBAR_DEFAULTS = /** @type {PreferenceDefinition[]} */ ([
   ["shortcuts.tabs", true],
   ["shortcuts.history", true],
   ["shortcuts.actions", true],
+  ["shortcuts.workspaces", true],
 
   // Boolean to determine if the providers defined in `exposureResults`
   // should be displayed in search results. This can be set by a
@@ -840,6 +844,8 @@ function makeDefaultResultGroups({
    */
   let rootGroup = {
     children: [
+      { children: [{ group: lazy.UrlbarUtils.RESULT_GROUP.ZEN_ACTION }] },
+      { children: [{ group: lazy.UrlbarUtils.RESULT_GROUP.ZEN_WORKSPACE }] },
       // heuristic
       {
         maxResultCount: 1,
